sentry_sdk/integrations/django/__init__.py

```diff
@@ -167,6 +168,7 @@ def setup_once():
 
         old_app = WSGIHandler.__call__
 
+        @functools.wraps(old_app)
```

Comment:
```suggestion
```

`@ensure_integration_enabled` already internally also decorates the function with `@functools.wraps`, so this should be unnecessary